docs/references/plugin-compatibility.md

```diff
@@ -0,0 +1,67 @@
+# Plugin Compatibility
+
+DB-less mode is the preferred choice for controller-managed Kong and Kong
+Enterprise clusters. However, not all plugins are available in DB-less mode.
+Review the table below to check if a plugin you wish to use requires a
+database.
+
+Note that some DB-less compatible plugins [have some limitations or require
+non-default configuration for
+compatibility](https://docs.konghq.com/latest/db-less-and-declarative-config/#plugin-compatibility).
+
+There are [two distributions of Kong Enterprise](./version-compatibility.md),
+`kong-enterprise-edition` and `kong-enterprise-k8s`. `kong-enterprise-k8s` is
```

Comment:
Let's instead use a link to https://github.com/Kong/kubernetes-ingress-controller/tree/master/docs/deployment